pkg/quota/controller/clusterquotamapping/clusterquotamapping.go

```diff
@@ -355,9 +355,9 @@ func (c *ClusterQuotaMappingController) syncQuota(quota *quotaapi.ClusterResourc
 }
 
 func (c *ClusterQuotaMappingController) syncNamespace(namespace *kapi.Namespace) error {
-	allQuotas, err := c.quotaLister.List(kapi.ListOptions{})
-	if err != nil {
-		return err
+	allQuotas, err1 := c.quotaLister.List(kapi.ListOptions{})
+	if err1 != nil {
+		return err1
```

Comment:
This does seem silly, but your method is ~35 lines long and has indentation five levels deep. Maybe make your `for{}` block a named function?